src/index.js

```diff
@@ -14,25 +13,31 @@ document.querySelector('#form-section').addEventListener('submit', (e) => {
 
   // Validation
   if (name === '' || points === '') {
-    // UI.showAlert('Please fill in all fields', 'danger', 1500);
+    UI.showAlert('Please fill in all fields', 'danger', 1500);
   } else {
+    // fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/', {
+    //   method: 'POST',
+    //   headers: {
+    //     'Content-Type': 'application/json',
+    //   },
+    //   body: JSON.stringify({
+    //     name: 'Duel of Fates',
+    //   }),
+    // })
+    //   .then((res) => res.json())
+    //   .then((data) => console.log(data.result));
```

Comment:
- [ ] I kindly suggest that you remove commented out code as it doesn't make your code look neat.

Reply:
- [ ] Comments removed